internal/manifests/config/tempo-config.yaml

```diff
@@ -257,6 +257,7 @@ usage_report:
   reporting_enabled: false
 query_frontend:
   search:
+    max_spans_per_span_set: 0
```

Comment:
Why do we need this setting? For the RBAC?

Reply:
It was a breaking change on Tempo